packages/utils/src/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from './internals/cheerio';
```

Comment:
can you describe this convention with `internals` folder? it seems like everything is an internal, yet we export everything, so... are they really internals? :]

i know there is one approach with two barrel files, one for public exports (index.ts) and one for all exports (internal.ts), which might be a way to get around import cycles, but never really used internals as a folder

Reply:
Well, in most places/modules I do, I use `src/lib` for the actual code, but `lib` is gitignored here, so I chose the next thing. We can change that, of course, if you'd like

Reply:
we can improve on that later, i would maybe just put the files in src directly. btw that gitignored lib folder sounds like some historical thing, we can definitely make it not ignored, i dont see why not